internal/txapp/txapp.go

```diff
@@ -34,10 +44,11 @@ func NewTxApp(db DB, engine common.Engine,
 			accounts:   make(map[string]*types.Account),
 			gasEnabled: GasEnabled,
 		},
-		signer:           signer,
-		chainID:          chainID,
-		GasEnabled:       GasEnabled,
-		extensionConfigs: extensionConfigs,
+		signer:            signer,
+		chainID:           chainID,
+		GasEnabled:        GasEnabled,
+		extensionConfigs:  extensionConfigs,
+		emptyVoteBodyTxSz: voteBodyTxSz,
```

Comment:
seems a bit unnecessary to do this here, since it is only used once. Maybe just calculate the emptyVoteBodyTxSize when you need it?

Also, a minor nit: I notice you do `sz` a lot for `size` (e.g. `emptyVoteBodyTxSz`, `eventSz`, etc.). I'd rather we just have the full word here, and have slightly longer variable names (especially if we are only saving two characters). Obviously there are times where abbreviation is good, but for something like this it seems overkill and could cause unnecessary confusion

Reply:
Since voteBodyTxSz is constant for a given chain ID so it would be a shame to compute it repeatedly since it involves a bunch of rlp enoding.  It would be computed each call to `ProposerTxs` I think.

Reply:
Ah that's a good point. Ok